extensions/rich_text_components/Image/directives/oppia-noninteractive-image.component.spec.ts

```diff
@@ -323,4 +323,13 @@ describe('NoninteractiveImage', () => {
       // This is tested to make sure the function did no continue to execute.
       expect(imagePreloaderService.getDimensionsOfImage).not.toHaveBeenCalled();
     });
+  it('should show alt text images when showAltText property is true', () => {
```

Comment:
Can you add a test for `showAltText = false` case?

Reply:
Done, thanks